client/services/helper.js

```diff
@@ -0,0 +1,63 @@
+import axios from 'axios';
+
+/**
+ *  Access JWT token from user's browser. If user has no token,
+ *  set equal to 'default' which the server will register as invalid
+ *  and redirect to login.
+ */
+var localStorageToken = local.storage.getItem('token') ? local.storage.getItem('token') : 'default';
```

Comment:
In my opinion, `window.localStorage` would be more better. [https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage](url)